src/heaps.jl

```diff
@@ -37,6 +37,9 @@
 #   - update!(h, i, v)    updates the value of an element
 #                         (referred to by the handle i)
 #
+#   - delete!(h, id)      deletes the node present at
+#                         node_index id from the heap
+#
 #   - top_with_handle(h)  return the top value of a heap
 #                         and its handle
 #
```

Comment:
For consistency with the other mutable heap functions, I think we need to change `delete!` to work on a `handle` instead of a `node_index`.

Reply:
In a minute!

Reply:
Done @kmsquire .